sonar-db/src/main/java/org/sonar/db/version/v62/PopulateOrganizationUuidOfUserRoles.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.version.v62;
+
+import java.sql.SQLException;
+import org.sonar.db.Database;
+import org.sonar.db.version.BaseDataChange;
+import org.sonar.db.version.MassUpdate;
+import org.sonar.db.version.Select;
+
+import static com.google.common.base.Preconditions.checkState;
+
+public class PopulateOrganizationUuidOfUserRoles extends BaseDataChange {
+
+  private static final String INTERNAL_PROPERTY_DEFAULT_ORGANIZATION = "organization.default";
+
+  public PopulateOrganizationUuidOfUserRoles(Database db) {
+    super(db);
+  }
+
+  @Override
+  public void execute(Context context) throws SQLException {
+    String organizationUuid = selectDefaultOrganizationUuid(context);
+
+    MassUpdate massUpdate = context.prepareMassUpdate();
+    massUpdate.select("select id from user_roles where organization_uuid is null");
```

Comment:
given that table groups can't be huge (at lest to my knowledge), I think it would be more efficient and not risky to do a single update rather than many